### ISSUE TITLE
Incorperate changes forks

### ISSUE THREAD

**Issue by witjoh** [eirc/puppet-single_user_rvm#12]
Hi, 
I incorperated a lot of changes from different forks, and added some testing.  If interested, I can create a PR for all those changes.  In the meantime, my forked repo can be found here : https://github.com/witjoh/puppet-single_user_rvm

Grts

**Comment by genebean** [comment]
@witjoh have you considered pushing your version to the Puppet Forge?

**Comment by witjoh** [comment]
@genebean Not yet.  I should refactor the module first for puppet 6.  A lot of improvements could be done I beleive.  On my TODO.  I will push it to to top of it.

**Comment by genebean** [comment]
Awesome. I am also going to send a PR or two your way. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by genebean** [comment]
Any objections to me doing some refactoring to simplify some of the code?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by witjoh** [comment]
not at all. Trigger me from time to time, so I can merge your PR's in short time frame :)

**Comment by genebean** [comment]
Will do. I need to verify I can use this in CentOS 7 with needing additional modifications and then I’ll submit my PR. (Debian 9 needed a couple of minor things)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
